source/loader/ze_loader_api.cpp

```diff
@@ -97,6 +97,7 @@ zelLoaderTranslateHandleInternal(
     }
 
     bool validHandle = false;
+    *handleOut = handleIn;
```

Comment:
don't we need nullptr check before dereferencing the pointer?

Reply:
I can add that at the start of this api.